app/javascript/widget/views/ArticleViewer.vue

```diff
@@ -1,16 +1,24 @@
 <script>
 import IframeLoader from 'shared/components/IframeLoader.vue';
+import { getLanguageDirection } from 'dashboard/components/widgets/conversation/advancedFilterItems/languages';
 
 export default {
   name: 'ArticleViewer',
   components: {
     IframeLoader,
   },
+  computed: {
+    isRTL() {
+      return this.$root.$i18n.locale
+        ? getLanguageDirection(this.$root.$i18n.locale)
+        : false;
+    },
+  },
 };
 </script>
 
 <template>
   <div class="bg-white h-full">
-    <IframeLoader :url="$route.query.link" />
+    <IframeLoader :url="$route.query.link" :is-rtl="isRTL" />
```

Comment:
I guess we need to pass the dark mode as well in the URL.

Reply:
@pranavrajs Fixed https://github.com/chatwoot/chatwoot/pull/11022/commits/571a342dae4008613361b4860d013af14f913134